src/bin/cli/logs/mod.rs

```diff
@@ -0,0 +1,37 @@
+mod compare;
+mod sentry;
+
+use catalyst_toolbox::logs::sentry::Error as SentryLogError;
+use structopt::StructOpt;
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error(transparent)]
+    SentryLogError(#[from] SentryLogError),
+
+    #[error(transparent)]
+    IoError(#[from] std::io::Error),
+
+    #[error(transparent)]
+    JsonError(#[from] serde_json::Error),
+
+    #[error(transparent)]
+    CompareError(#[from] compare::Error),
+}
+
+#[derive(StructOpt)]
+#[structopt(rename_all = "kebab-case")]
+pub enum Logs {
+    Sentry(sentry::SentryLogs),
+    Compare(compare::Compare),
```

Comment:
Please add documentation / help strings to explain what the commands do.

Reply:
Absolutely